### PR TITLE
Documentation update for GraalVM 0.21

### PR DESCRIPTION
**Pull request by pitr-ch**
Not to be merged before 0.21 is avaliable.

### CODE REVIEW

**Comment by chrisseaton** [comment]
These docs get shipped with GraalVM though. They could go in now and we could update the GraalVM PR (might be too late). People looking at code in master can't expect it to be correct for whatever version they have locally.

Thanks for this great work on supporting version managers though!

File: doc/user/ruby-managers.md [hunk 1]
```diff
@@ -1,8 +1,40 @@
...
+
+RVM has a command for adding precompiled Ruby.
+
+```bash
+rvm mount path/to/graalvm/language/ruby -n truffleruby
+```
+
+## rbenv
+
+To add TruffleRuby to rbenv a symbolic link has to be added to the `versions`
```
Comment (eregon) [inline doc/user/ruby-managers.md]: `To add TruffleRuby to rbenv, `

File: doc/user/ruby-managers.md [hunk 2]
```diff
@@ -1,8 +1,40 @@
...
+# Configuring Ruby managers
 
-If you are using a Ruby manager like `rvm`, `rbnev`, or `chruby` make 
-sure that the manager does not set environment variables `GEM_HOME`, `GEM_PATH`,
-and `GEM_ROOT`. The variables are picked up by truffleruby (as any other Ruby 
+It's recommended to add TruffleRuby to a Ruby manager for ease of use.
+
+## RVM
+
+RVM has a command for adding precompiled Ruby.
```
Comment (eregon) [inline doc/user/ruby-managers.md]: RVM has a command for adding a precompiled Ruby to the list of available rubies:

File: doc/user/ruby-managers.md [hunk 3]
```diff
@@ -1,8 +1,40 @@
...
+## chruby
+
+To add TruffleRuby to chruby a symbolic link has to be added to the `$HOME/.rubies` 
+directory.
+
+```bash
+ln -s path/to/graalvm/language/ruby "$HOME/.rubies/truffleruby"
+```
+
+## Using TruffleRuby without Ruby manager
```
Comment (eregon) [inline doc/user/ruby-managers.md]: without `a`

File: doc/user/ruby-managers.md [hunk 4]
```diff
@@ -1,8 +1,40 @@
...
+To add TruffleRuby to `chruby` a symbolic link has to be added to the `$HOME/.rubies` 
+directory:
+
+```bash
+ln -s path/to/graalvm/language/ruby "$HOME/.rubies/truffleruby"
+```
+
+## Using TruffleRuby without a Ruby manager
+
+If you are using a Ruby manager like `rvm`, `rbnev`, or `chruby` and wish
```
Comment (eregon) [inline doc/user/ruby-managers.md]: `rbenv`

**Pull request merged**